bin/sl-deploy.js

```diff
@@ -5,18 +5,21 @@
 var Parser = require('posix-getopt').BasicParser;
 var debug = require('debug')('strong-deploy');
 var defaults = require('strong-url-defaults');
-var getPackageInfo = require('../lib/package').getPackageInfo;
 var defaultPackagePath = require('../lib/package').getPackagePath;
 var deploy = require('../');
 var fs = require('fs');
+var getPackageInfo = require('../lib/package').getPackageInfo;
+var glb = require('strong-globalize');
 var maybeTunnel = require('strong-tunnel');
 var path = require('path');
 
-function printHelp($0, prn) {
-  var USAGE = fs.readFileSync(require.resolve('./sl-deploy.txt'), 'utf-8')
-      .replace(/%MAIN%/g, $0)
-      .trim();
+glb.setRootDir(path.resolve(__dirname, '..'));
+glb.setDefaultLanguage();
 
+function printHelp($0, prn) {
+  var USAGE = glb.t('sl-deploy.txt')
```

Comment:
just used heuristics: key ended with ".txt" is considered as a file which is then searched in the intl/<lang> directory, stringified, and used as a long value.